docs/section10/10-1.md

```diff
@@ -0,0 +1,41 @@
+# Airside systems – Rule 10-1  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False    
+**Rule ID:** 10-1  
+ 
+**Rule Description:** The proposed design includes humidification (which does not comply with 90.1-2019 Section 6.5.2.4) and the baseline building design has been modeled with adiabatic humidification, as expected.  
+
```

Comment:
I suggest that we aggregate 10-1 and 10-2, to avoid having to differentiate between systems in the proposed design that do and do not comply with Section 6.5.2.4.

Reply:
Done